juniper_official/Chassis/chassis-alarms.rule

```diff
@@ -0,0 +1,91 @@
+/*
+ * Detects chassis alarms and notifies when anomalies are found.
+ *  
+ */
+ healthbot {
+    topic chassis.alarms {
+        rule check-chassis-alarms {
+            synopsis "Chassis alarms detector";
+            description "Collects chassis stats and notify anomalies when alarms found";
+	        /*
+            * Use alarm-description slot as key for rule.
+            */
+            keys alarm-description;
+            sensor chassis {
+			    synopsis "Chassis netconf alarm sensor";
+                description "Chassis iAgent sensor to collect data from network device";
+                iAgent {
+                    file Chassis-Alarms.yml;
+                    table ChassisAlarmTable;
+                    frequency 60s;
+                }
+            }
+            field alarm-description {
+                sensor chassis {
+                    path alarm-description;
+                    data-if-missing {
+                        value "No alarms currently active";
+                    }
+                }
+                type string;
+                description "Chassis alarm description";
+            }
+            /*
+            * Anomaly detection logic for buffer memory utilization.
+            */
+            trigger chassis-alarms {
+                synopsis "Chassis alarms KPI";
+                description "Sets health based on chassis alarm type";
+                frequency 60s;
+                /*
+                * Sets color to red and sends out an anomaly notification when
+                * major chassis alarms detected
+                * 
+                */
+                term is-major-alarm-exists {
```

Comment:
Please change this to does-major-alarm-exist.

Reply:
Corrected